api/android/build-android-lib.sh

```diff
@@ -30,28 +31,28 @@
 # 'lite' : with GStreamer core plugins
 # 'single' : no plugins, single-shot only
 # 'internal' : no plugins, single-shot only, disable SNAP and tf-lite
-build_type='all'
+build_type="all"
 
-nnstreamer_api_option='all'
-include_assets='no'
+nnstreamer_api_option="all"
+include_assets="no"
 
-# Set target ABI ('armeabi-v7a', 'arm64-v8a', 'x86', 'x86_64')
-target_abi='arm64-v8a'
+# Set target ABI ('armeabi-v7a', 'arm64-v8a')
+target_abi="arm64-v8a"
 
 # Run instrumentation test after build procedure is done
-run_test='no'
+run_test="no"
 
 # Variables to release library (GROUP:ARTIFACT:VERSION)
-release_bintray='no'
+release_bintray="no"
 
 # Enable SNAP
-enable_snap='yes'
+enable_snap="yes"
 
 # Enable tensorflow-lite
-enable_tflite='yes'
+enable_tflite="yes"
 
 # Set tensorflow-lite version (available: 1.9 and 1.13)
-nnstreamer_tf_lite_ver='1.13'
+nnstreamer_tf_lite_ver="1.13"
```

Comment:
Be prepared to add 1.15.2 soon (for next version, mid-April).
The next reference TF-Lite is 1.15.2 (using the forked one from tizen.org)

Reply:
Now set tf-lite version 1.15.2